### PR TITLE
Add convention comparison chart to Introduction to 3D

### PR DESCRIPTION
**Pull request by Calinou**
This is useful information when porting code or assets from another 3D engine. The chart was created by [Freya Homlér](https://mobile.twitter.com/FreyaHolmer).

See https://github.com/godotengine/godot-docs/issues/6272.

### CODE REVIEW

File: tutorials/3d/introduction_to_3d.rst [hunk 1]
```diff
@@ -137,15 +137,19 @@ floating-point precision issues (and thus, glitches or artifacts) in
...
+The Y coordinate is used for "up". As for the horizontal X/Z axes, Godot uses a
+**right-handed** coordinate system. This means that for most objects that need
+alignment (such as lights or cameras), the Z axis is used as a "pointing
+towards" direction. This convention roughly means that:
 
 -  **X** is sides
 -  **Y** is up/down
 -  **Z** is front/back
 
+See this chart for comparison with other 3D engines:
```
Comment (skyace65) [inline tutorials/3d/introduction_to_3d.rst:149]:
```suggestion
See this chart for comparison with other 3D software:
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
1 minor issue but this looks good. Also I know we're debating moving to webp for images and one of the potential issues is making the repository bigger with format conversions. Since this is a brand new image should it be webp since it avoids that issue?

**Comment by mhilbrunner** [comment]
> we're debating moving to webp
We should come to a decision on this, but I think moving to .webp for _new_ images makes sense at this point.

**Comment by fire** [comment]
Did we have a note of permission for Freya's image?

**Comment by Calinou** [comment]
> Did we have a note of permission for Freya's image?

No, but feel free to ask her if you find a way to.

**Comment by mhilbrunner** [comment]
> > Did we have a note of permission for Freya's image?
> 
> No, but feel free to ask them if you find a way to.

Reached out to ask for permission, which we definitely should get before/if we add this. It's a great graphic though.

**Comment by mhilbrunner** [comment]
Freya gave the OK on using it and licensing it as CC BY, as long as its with attribution and a link to https://twitter.com/FreyaHolmer

**Comment by Calinou** [comment]
> Freya gave the OK on using it and licensing it as CC BY, as long as its with attribution with name + a link to [twitter.com/FreyaHolmer](https://twitter.com/FreyaHolmer)

Done.

**Review by fire — APPROVED** [review]

**Review by skyace65 — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Pushed the .webp update. Thanks for the PR & merge :)

**Pull request merged**